pkg/manager/manager.go

```diff
@@ -85,6 +85,7 @@ func New(opts Options) (Manager, error) {
 	controllerManagerContext := &context.ControllerManagerContext{
 		Context:                 goctx.Background(),
 		Namespace:               opts.WatchNamespace,
```

Comment:
This looks like a bug. Should be `WatchNamespace:               opts.WatchNamespace,` as far as I can tell.